.eslintrc.yml

```diff
@@ -35,10 +35,6 @@ rules:
 
   default-case:
     - error
-  indent:
-    - error
-    - 2
-    - SwitchCase: 1
```

Comment:
This was causing problems with functions where the return type was an object, like:

```js
export function promiseWithResolvers<T>(): {
  promise: Promise<T>;
  resolve: (value: PromiseLike<T> | T) => void;
  reject: (reason?: any) => void;
} {   // <- eslint said there was an error here, but that's how prettier auto-formats it
  // Code here
}
```